l5kit/Pipfile

```diff
@@ -0,0 +1,36 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+mypy = "*"
+pytest-cov = "*"
+flake8 = "*"
+isort = "*"
+sphinx = "*"
+sphinx-rtd-theme = "*"
+recommonmark = "*"
+pre-commit = "*"
+black = "==19.10b0"
+importlib-metadata = "*"  # required for python 3.8
+
+[packages]
+zarr = "*"
+numpy = "*"
+opencv-contrib-python-headless = "*"
+pymap3d = "*"
+scipy = "*"
+tqdm = "*"
+transforms3d = "*"
+pyyaml = "*"
+matplotlib = "*"
+imageio = "*"
+notebook = "*"
+ptable = "*"
+ipywidgets = "*"
+protobuf = ">=3.12.2"
+torch = ">=1.5.0,<1.6.0"
+torchvision = ">=0.6.0,<0.7.0"
+l5kit = {editable = true, path = "."}
```

Comment:
```suggestion
l5kit = {editable = true, path = "./l5kit"}
```

Reply:
I'm not able to `pipenv lock` anymore if I add this:
`pipenv.vendor.requirementslib.exceptions.RequirementError: Failed parsing requirement from './l5kit'` 
 
Why should it point to `./l5kit` and not `.` ?

Reply:
I think [it should point](https://stackoverflow.com/questions/53378416/what-does-pipenv-install-e-do-and-how-to-use-it) to where the `setup.py` is (otherwise pip doesn't know how to build it). If that's the case it makes sense to leave it as `.` here, what do you think @giacomodabisias

Reply:
Ah your pipfile is in the L5kit folder, I see. Usually you have the pipfile and pipfile.lock one folder up, since you might create new packages in the same repo and you can use the pipfile to manage all of them if they work together.
I suggest moving the files up one folder to the main folder of the repo

Reply:
mmm...given that this is not the case for this repo (not planned anyway) I would leave it there. The reason is also that all other configuration files (and `setup.py` itself) are already placed there. Moving only this one one level up would create some confusion about where to run what imo (e.g. install and add packages vs running tests, lint or installing from `setup.py`).

Reply:
up to you :) I am fine also with the current structure.
Notice though that there would be no confusion since tests, linting etc..are all independent from the file location.
Setup.py should be in the package as it is now and should not be moved :)

Reply:
> Notice though that there would be no confusion since tests, linting etc..are all independent from the file location.

They still need to read their config files though, otherwise they run with default configs.

I'll leave it as it is for now but I'll consider it for the future :) Can I have your final blessing @giacomodabisias 😄 ?

Reply:
sure, go for it